com.unity.netcode.gameobjects/Samples/Bootstrap/Scripts/BootstrapManager.cs

```diff
@@ -0,0 +1,67 @@
+﻿using UnityEngine;
+
+namespace Unity.Netcode.Samples
+{
+    /// <summary>
+    /// Class to display helper buttons and status labels on the GUI, as well as buttons to start host/client/server.
+    /// Once a connection has been established to the server, the local player can be moved via a GUI button.
+    /// </summary>
+    public class BootstrapManager : MonoBehaviour
+    {
+        void OnGUI()
+        {
+            GUILayout.BeginArea(new Rect(10, 10, 300, 300));
+            if (!NetworkManager.Singleton.IsClient && !NetworkManager.Singleton.IsServer)
+            {
+                StartButtons();
+            }
+            else
+            {
+                StatusLabels();
+
+                // move button will only be displayed on clients
+                if (NetworkManager.Singleton.IsClient)
+                {
+                    SubmitNewPosition();
+                }
+            }
+
+            GUILayout.EndArea();
+        }
+
+        static void StartButtons()
+        {
+            if (GUILayout.Button("Host")) NetworkManager.Singleton.StartHost();
+            if (GUILayout.Button("Client")) NetworkManager.Singleton.StartClient();
+            if (GUILayout.Button("Server")) NetworkManager.Singleton.StartServer();
+        }
+
+        static void StatusLabels()
+        {
+            var mode = NetworkManager.Singleton.IsHost ?
+                "Host" : NetworkManager.Singleton.IsServer ? "Server" : "Client";
+
+            GUILayout.Label("Transport: " +
+                NetworkManager.Singleton.NetworkConfig.NetworkTransport.GetType().Name);
+            GUILayout.Label("Mode: " + mode);
+        }
+
+        static void SubmitNewPosition()
+        {
+            if (GUILayout.Button("Move Local Player"))
```

Comment:
We have NetworkTransform for movement, it feels weird to use RPCs to "move local player" instead of the dedicated type we provide for doing this...

Maybe NetworkTransform to move the player, and then a button for "shooting" or spawning or some other RPC event?

@SamuelBellomo thoughts?

Reply:
it is not actually moving the object, it is asking server to teleport us to a random position on the server side. so the actual networktransform movement/replication is still handled by the networktransform component itself.

FWIW, we might make method names better though.

Reply:
+1 Fatih
@fernando-cortez we could use "RandomMoveServerRpc"? Or "MovePlayerRandomServerRpc"?
I also added a small comment suggestion improvement to clarify that NetworkTransform is the one replicating to other clients